include/crm/pengine/internal.h

```diff
@@ -179,12 +179,8 @@ enum pe_warn_once_e {
 extern uint32_t pe_wo;
 
 #define pe_warn_once(pe_wo_bit, fmt...) do {    \
-        if (!pcmk_is_set(pe_wo, pe_wo_bit)) {  \
-            if (pe_wo_bit == pe_wo_blind) {     \
-                crm_warn(fmt);                  \
-            } else {                            \
-                pe_warn(fmt);                   \
-            }                                   \
+        if (!pcmk_is_set(pe_wo, pe_wo_bit)) {   \
+            pe_warn(fmt);                       \
```

Comment:
We do still want to special-case pe_wo_blind, because we don't want crm_verify to fail a configuration with that condition (and it doesn't need to show up in the crm_verify XML output)